### PR TITLE
Start the Kafka migration series by Christian

### PR DESCRIPTION
**Pull request by rtyler**
This has already been copy-edited and proofed from the internal wiki version written by @xianwill

### CODE REVIEW

**Comment by rtyler** [comment]
NOTE TO REVIEWERS: This should not be published before Dec 26th

File: _posts/2019-12-26-migrating-to-kafka-in-aws.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+empowering, and mission-critical. In order to realize this future, we had to
+move from our deprecated Kafka deployment, to a newer well-managed Apache
+Kafka environment in the cloud.
+
+When I joined Scribd earlier this year, Kafka existed, and that's about it. We
+weren't using it to its potential.  Part of the "Kafka avoidance" syndrome that
+existed, stemmed from the operational difficulties of _just_ running the thing.
+It was almost like we were afraid to touch Kafka for fear it might fall over.
+Another part of that avoidance grew out of the functionality not matching
+developers' expectations developer expectations.  When we first adopted Kafka,
```
Comment (xianwill) [inline _posts/2019-12-26-migrating-to-kafka-in-aws.md]: small repeated phrase typo here `developers' expectations developer expectations.`

File: _posts/2019-12-26-migrating-to-kafka-in-aws.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+for capacity planning and vendor evaluation, since it gave us a way to see how
+the enrichments we desired in our pipeline might impact the overall load and
+storage demands on the cluster.  It also gave us some perspective on the
+complexity of operating multiple streaming applications to deliver higher
+quality data in real-time.
+
+### Vendor Evaluation
+
+Determining the vendor for the future of your streaming data platform is no
+light-hearted decision! For each vendor, we went created the topics and ACLs necessary
```
Comment (xianwill) [inline _posts/2019-12-26-migrating-to-kafka-in-aws.md]: we ~went~ created

File: _posts/2019-12-26-migrating-to-kafka-in-aws.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+* **How well can we secure and manage it?** Our
+  legacy on-premise deployment was wide-open internally 😱. Client applications
+  did not need to authenticate to write messages to a specific topic. They only
+  needed network access. In our new cluster, we wanted to make sure each client
+  was constrained by ACLs (Access Control Lists).
+
+### Prototype Approach
+
+To get some hand-on experience with each platform, we implemented a prototype
+to represent one of our larger analytics workloads. Not only was it a big data stream to work with, but it also would benefit from some extensions we could imagine for the future, such as message debatching, routing and validation streams. The
```
Comment (xianwill) [inline _posts/2019-12-26-migrating-to-kafka-in-aws.md]: stray line break

**Comment by rtyler** [comment]
Welp, I forgot to publish this yesterday. I think Monday of next week will be a good time to publish it instead.

**Pull request merged**